docs/basics/part-6-using-docker-with-earthly.md

```diff
@@ -148,7 +148,7 @@ When we use the `--compose` flag, Earthly will start up the services defined in
 You'll need to use `--allow-privileged` (or `-P` for short) to run this example. 
 
 ```bash
-earthly --allow-privileged +integration-tests
+earthly -P +integration-tests
```

Comment:
nit: personally I also use `-P` because it's shorter and easier.
But when I used to use earthly in a CI, I tried to use the long argument version, because I thought it was more comprehensible to others who might be less familiar with earthly